task/02-numbers-tasks.js

```diff
@@ -221,7 +227,8 @@ function isPrime(n) {
  *   toNumber(new Number(42), 0) => 42
  */
 function toNumber(value, def) {
-    throw new Error('Not implemented');
+    let result = Number(value);
+    return Number.isNaN(result) ? Number(def) : result;
```

Comment:
Yes, test have passed, but task condition:
otherwise returns **default** value passed as a second argument
and you've converted it into number